examples/generate.sh

```diff
@@ -121,3 +125,5 @@ echo "WARNING: ${PROVIDER_COMPONENTS_GENERATED_FILE} includes AWS credentials"
 
 # Patch kubernetes version
 sed -i'' -e 's|kubernetesVersion: .*|kubernetesVersion: '$KUBERNETES_VERSION'|' examples/_out/controlplane.yaml
+
+echo "NOTE: Ensure that the cert-manager components are running before creating the provider-components, cluster and control-plane."
```

Comment:
This should either link the quick start instructions or some other pre-req document, wdyt?

Reply:
That's a great idea. This issue is open currently (https://github.com/kubernetes-sigs/cluster-api/issues/1561). Once this is done I can update this script with the relevant link. But I don't think this should be a blocker.